test/integration/scheduler_perf/README.md

```diff
@@ -129,3 +129,25 @@ The test cases labeled as `short` are executed in pull-kubernetes-integration jo
 | ci-kubernetes-integration-master | integration-test       |
 | pull-kubernetes-integration      | integration-test,short |
 | ci-benchmark-scheduler-perf      | performance            |
+
+## Scheduling throughput thresholds
+
+Thresholds are used to capture scheduler performance regressions in a periodic ci-benchmark-scheduler-perf job. Most test cases have a threshold set for the largest `performance` workloads. By default, these are defined for the `Average` statistic of the `SchedulingThroughput` metric. It is possible to use other metric by configuring `thresholdMetricSelector` per test case or workload. 
```

Comment:
nit: please wrap lines. That might make future edits less intrusive.

Reply:
Good idea, changed